From 2714eb4523b74e068052883181c987a0a60753e3 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 8 Jul 2019 09:53:22 +0000
Subject: [PATCH] joyent/node-zkstream#9 use var instead of const so we can run
 in strict mode joyent/node-zkstream#29 unreliable tests assuming after sync()
 all watchers have fired joyent/node-zkstream#24 remove watchers from index
 once all event handlers are gone

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
-- 
2.21.0

